pkg/generate/config/config.go

```diff
@@ -217,6 +224,18 @@ func (c *Config) IsIgnoredShape(shapeName string) bool {
 	return util.InStrings(shapeName, c.Ignore.ShapeNames)
 }
 
+// OverrideValues gives list of member values to override.
+func (c *Config)  OverrideValues(operationName string) (map[string]string, bool) {
```

Comment:
there's an extra space in between `*Config)` and `OverrideValues(` :) Please run `go fmt` on your Go code to catch these little things; alternately, enable an auto-go-fmt plugin for your IDE or editor.

Reply:
Ah very nice catch! Enabled this in IDE thank you!